test/parser/test_parser.py

```diff
@@ -303,6 +291,26 @@ def test_pydantic_schema_validator_success(
                 "season": "autumn"  # ❌ Value that doesn't match any of the enum variants
             },
         ),
+        (
+            schemas.document_schema,
+            {
+                "title": None,
+                "document_url": None,  # ❌ Do not allow objects with all null values
+            },
+        ),
+        (
+            schemas.documents_schema,
+            {
+                "documents": []  # ❌ Do not allow objects with all null arrays
+            },
+        ),
+        (
+            schemas.document_schema,
+            {
+                "title": "",  # ❌ Do not allow objects with all empty strings
+                "document_url": "",
```

Comment:
might be good to add a test for an object with a list of empty dicts eg: {list: [{}, {a: None, b: {}, c: []]}